workings

```diff
@@ -0,0 +1,34 @@
+User story 1
+```
+As a Maker
+So that I can let people know what I am doing
+I want to post a message (peep) to chitter
+```
+create peep
+- write a feature test that:
+ visits '/' 
+ expects to see a text field 
+ expects to see a submit button
+post peep
+- write a feature test that:
+  visits '/'
+  fill_in placeholder with peep message
+  click_button 'post peep'
+  expects to see message/peep
+
+
+- write a feature test that:
+  visits '/'
+  click_button 'signup'
+  fill_in in placeholder with full name
+  fill_in placeholder with username
+  fill_in placeholder with email address
+  fill_in placeholder with password
+  click_button submit
+  redirect '/'
+  expects to see content 'Welcome to Chitter'
+
+CREATE TABLE post(id SERIAL PRIMARY KEY, peep VARCHAR (280), date_posted TIMESTAMP NOT NULL);
+INSERT INTO post (peep, date_posted) VALUES ('hello world', current_timestamp);
```

Comment:
Liked that you included the queries to make the tables. From how i understood it i think you could make both tables on one database. So you didn't need to make the account and account_test database. You could have had the signup and post tables in the chitter and chitter_test database. That being siad. It did work fine but might save you some extra time in the future.